.github/workflows/build-gtk.yml

```diff
@@ -0,0 +1,39 @@
+name: Build Controls.Sample.Gtk
+on: [push, pull_request]
+
+jobs:
+  build_and_test:
+    name: Build & Test
+    runs-on: ubuntu-22.04
+    env:
+      GtkSharpVersion: 3.24.24.95
+      DotnetVersion: 6.0.400
```

Comment:
I think we must update this after merge https://github.com/jsuarezruiz/maui-linux/pull/66

Reply:
this is done in [lytico/merge/main240321](https://github.com/lytico/maui/tree/lytico/merge/main240321)
with a net8-version of GtkSharp (currently -develop), see: https://github.com/lytico/maui/blob/bf56fbbdc417fc8958e5ca58711890f5db1ee19a/.github/workflows/build-gtk.yml#L9-L11